sssom/parsers.py

```diff
@@ -596,6 +639,8 @@ def _cell_element_values(cell_node, prefix_map: PrefixMap) -> Optional[Mapping]:
             except NoCURIEException as e:
                 logging.warning(e)
 
+    mdict["match_type"] = "Unspecified"
```

Comment:
Put "Unspecified" into constants.py. Generally, we should avoid using raw strings as keys in dictionaries, better use CONSTANTS like MATCH_TYPE_UNSPECIFIED. No need to change the whole code now :P

Reply:
Done